# - Battery data
int8 battery_low
uint8 battery_percentage
int16 battery_time_left

# - IMU data
int16 height
int16 north_speed
int16 east_speed
int16 ground_speed

# - States
int8 imu_calibration_state
int8 imu_state
int8 power_state
int8 camera_state
int8 battery_state

# - Stats 
int8 fly_mode
int8 fly_time
int8 drone_hover

# - WiFi data
int8 wifi_disturb
int8 wifi_strength
